moto/s3/notifications.py

```diff
@@ -89,6 +103,9 @@ def send_event(
     if bucket.notification_configuration is None:
         return
 
+    if not S3NotificationEvent.is_event_valid(event_name):
```

Comment:
This doesn't seem necessary - we always control when `send_event` is invoked, so there's no point in raising an exception